database/migrations/2022_01_13_123409_add_uuid_to_topic_scorm_scos_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddUuidToTopicScormScosTable extends Migration
```

Comment:
this migration is not needed, when we have sco `id` we can fetch `uuid` as model attribute